web/themes/new_weather_theme/templates/partials/hourly-table.html.twig

```diff
@@ -33,7 +33,9 @@
         {% for period in hours %}
         <td>
           {% if period.temperature is not same as (null) %}
-          {{ period.temperature }}<span class="font-sans-3xs text-ttop padding-left-1px">℉</span>
+          {{ period.temperature }}
+          <span class="usa-sr-only">℉</span>
```

Comment:
Could we remove the space between the number and the degree symbol? I believe it's because the `span` elements are on their own line?

**PR** 
![Screenshot of the gap](https://github.com/weather-gov/weather.gov/assets/150970973/b6c77b16-2538-4fc1-ae93-2a325baa935e)


**Goal**
![image](https://github.com/weather-gov/weather.gov/assets/150970973/3d3fe0ed-e7dd-4d70-89ff-f7ca2ea356a7)

Reply:
Oops! I knew something seemed off but I couldn't identify it. Yes, I'll make that fix!